AsyncDisplayKit/Layout/ASStackLayoutDefines.h

```diff
@@ -67,3 +67,23 @@ typedef NS_ENUM(NSUInteger, ASStackLayoutAlignSelf) {
   /** Expand to fill cross axis */
   ASStackLayoutAlignSelfStretch,
 };
+
+/** Orientation of children along horizontal axis */
+typedef NS_ENUM(NSUInteger, ASHorizontalAlignment) {
+  /** Left aligned */
+  ASAlignmentLeft,
+  /** Right aligned */
+  ASAlignmentRight,
+  /** Center aligned */
+  ASAlignmentMiddle,
```

Comment:
Small detail, but let's put this in the middle of the three just like Center below :)